tools/github-actions/nanomap-renderer-invoker.sh

```diff
@@ -9,7 +9,7 @@ cd "data/nanomaps"
 mv "cyberiad_nanomap_z1.png" "Cyberiad_nanomap_z1.png"
 mv "delta_nanomap_z1.png" "Delta_nanomap_z1.png"
 mv "MetaStation_nanomap_z1.png" "MetaStation_nanomap_z1.png"
-mv "CereStation_nanomap_z1.png" "CereStation_nanomap_z1.png"
+mv "cereStation_nanomap_z1.png" "CereStation_nanomap_z1.png"
```

Comment:
```suggestion
mv "cerestation_nanomap_z1.png" "CereStation_nanomap_z1.png"
```

Its in the code as `_maps/map_files/cerestation/cerestation.dmm`